docs/user-guide/overview.md

```diff
@@ -12,9 +12,16 @@ For all organizations the storage panel displays the total number of archived it
 
 ## Crawling
 
+The crawling panel lists the number of currently running and waiting crawls, as well as the total number of pages captured.
+
+### Execution Time
+
+`Paid Feature`{.badge-green}
+
 For organizations with a set execution minute limit, the crawling panel displays a graph of how much execution time has been used and how much is currently remaining. Monthly execution time limits reset on the first of each month at 12:00 AM GMT.
 
-The crawling panel also lists the number of currently running and waiting crawls, as well as the total number of pages captured.
+??? Question "How is execution time calculated?"
+    Execution time is calculated by multiplying a crawl's elapsed time by the [_Crawler Instances_](../workflow-setup/#crawler-instances) scale value. Like elapsed time, this is tracked as the crawl runs and lowering the _Crawler Instances_ scale while a crawl is running will also reduce the execution time multiplier.
```

Comment:
Hm, this is an approachable definition but not quite a correct one I think.

MIght I suggest:

```suggestion
Execution time is the total amount of time all [_Crawler Instances_](../workflow-setup/#crawler-instances) are actively running during a crawl. For instance, if _Crawler Instances_ scale is set to 2x and each crawler instance uses 2 minutes of active crawling time, execution time for the crawl will be 4 minutes. Like elapsed time, this is tracked as the crawl runs. Lowering the _Crawler Instances_ scale while a crawl is running will typically also reduce the amount of execution time used.
```

Reply:
Probably not quite there, but closer. We don't multiple elapsed time by crawl scale because we don't include time that the crawler pods are e.g. spinning up or restarting as part of the execution time, only when they are actively running.

Reply:
Ah good catch!  Have made some minor wording changes, will accept the suggestion?

Reply:
> ...Though thinking about this again, I'm not sure how important the explanation is. Won't the crawl more or less use the same amount of execution time either way?

Not necessarily! It really depends on how many workers there are, how much time the crawler spends per page, and so on. Usually higher crawl scale means less elapsed time but more execution time, but we can't really say with certainty because sometimes it doesn't work out that way.

Reply:
It stays!